src/components/KTreeList/KTreeList.vue

```diff
@@ -0,0 +1,456 @@
+<template>
+  <div
+    :class="{ 'dragging': dragging }"
+    class="k-tree-list"
+    @keydown.esc="outOfBounds = true"
+  >
+    <draggable
+      v-bind="draggableAttrs"
+      :disabled="disableDrag"
+      :list="internalList"
+      :group="{ name: 'k-tree-list', pull: !outOfBounds, put: !maxLevelReached && !outOfBounds, revertClone: outOfBounds }"
+      :move="checkMove"
+      :level="level"
+      :sort="false"
+      @start="onStartDrag"
+      @end="onStopDrag"
+      @change="handleChangeEvent"
+    >
+      <template #item="{ element }">
+        <div
+          :style="indentStyle"
+          :class="{
+            'non-root': level !== 0
```

Comment:
Curious why we need this `non-root` class?

Reply:
So that we don't get the connector lines that child items have to their parents

Reply:
You can target the immediate children using the `>` operator and do something like this: 
`.k-tree-select > .k-tree-item` and set `display: none` for any pseudo selectors.

Reply:
We can't use this because we create an instance of `KTreeList` inside of `KTreeList` and those are the nested children. This means that there will be `.k-tree-list ... .k-tree-list` instances that we sometimes need to apply to.

Reply:
We should be able to have a wrapper element for the entire KTree. May be giving a class name to that element will give us an option to only select the L1 items. So you can do:
```
.k-tree-wrapper > .[immediate-child-class-name]
```

If you want a quick reference: https://codesandbox.io/s/eloquent-shamir-op101f